### PR TITLE
Fix ESLint

### PR DESCRIPTION
**Pull request by felixfbecker**
ESLint wasn't even running in CI, that's why my previous PR passed 🤦‍♂ 
This fixes the remaining cases that weren't fixable with `--fix`.

### CODE REVIEW

File: extensions/go/src/xrefs.ts [hunk 1]
```diff
@@ -67,7 +70,7 @@ export function createExternalReferencesProvider({
         }
 
         // TODO - support named imports
-        return findReposViaSearch(`file:\.go$ \t"${packageName}" max:${limit}`)
+        return findReposViaSearch(`file:\\.go$ \t"${packageName}" max:${limit}`)
```
Comment (felixfbecker) [inline extensions/go/src/xrefs.ts]: Caught a legit bug!

**Comment by felixfbecker** [comment]
@sourcegraph/code-intel ready for review

File: extensions/go/src/xrefs.ts (hunk 1 above)
Comment (eseliger) [reply extensions/go/src/xrefs.ts]: robots take over the world!

**Review by eseliger — APPROVED** [review]

**Comment by efritz** [comment]
Please wait for a code owner approval before merging.

**Pull request merged**